packages/kbn-bazel-packages/src/bazel_package_dirs.ts

```diff
@@ -21,13 +21,15 @@ import { REPO_ROOT } from '@kbn/utils';
  */
 export const BAZEL_PACKAGE_DIRS = [
   'packages',
-  'packages/shared-ux',
-  'packages/shared-ux/*',
   'packages/analytics',
   'packages/analytics/shippers',
   'packages/analytics/shippers/elastic_v3',
   'packages/core/*',
   'packages/home',
+  'packages/kbn-content-management',
+  'packages/kbn-content-management/*',
```

Comment:
I don't this this line is necessary, packages will be found that are directly within `packages/kbn-content-management` will be found thanks to the previous line.

Reply:
Ok I will remove it 👍